.zuul.yaml

```diff
@@ -107,6 +110,9 @@
       cifmw_update_containers_openstack: false
       deploy_watcher_service_extra_vars:
         watcher_cr_file: "ci/watcher_v1beta1_watcher_tlse.yaml"
+      cifmw_edpm_telemetry_enabled_exporters:
+        - podman_exporter
+        - openstack_network_exporter
```

Comment:
I think this var will go under *watcher-operator-validation-base* job.

Reply:
https://github.com/openstack-k8s-operators/watcher-operator/pull/201 will cleanup the old job.

Reply:
Thanks